stellar_contract_sdk_macros/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "stellar_contract_sdk_macros"
+version = "0.0.1"
+authors = ["Stellar Development Foundation <info@stellar.org>"]
+license = "Apache-2.0"
+edition = "2021"
+
+[lib]
+proc-macro = true
```

Comment:
Seems to cause https://github.com/rust-lang/rust/issues/93988 on macos, but probably an insignificant issue.